--- a/Makefile
+++ b/Makefile
@@ -15,8 +15,8 @@
 INSTALL_INC= $(INSTALL_TOP)/include
 INSTALL_LIB= $(INSTALL_TOP)/lib
 INSTALL_MAN= $(INSTALL_TOP)/man/man1
-INSTALL_LMOD= $(INSTALL_TOP)/share/lua/$V
-INSTALL_CMOD= $(INSTALL_TOP)/lib/lua/$V
+INSTALL_LMOD= $(INSTALL_TOP)/share/lua/$(VER)
+INSTALL_CMOD= $(INSTALL_TOP)/lib/lua/$(VER)
 
 # How to install. If your install program does not support "-p", then
 # you may have to run ranlib on the installed liblua.a.
@@ -45,8 +45,8 @@
 TO_MAN= lua.1 luac.1
 
 # Lua version and release.
-V= 5.2
-R= $V.4
+VER= 5.2
+R= $(VER).4
 
 # Targets start here.
 all:	$(PLAT)

